basic/instCountTest.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2016, 2017 National University of Singapore
```

Comment:
@rasoolmaghareh I usually won't put a copyright information on code where we reuse parts of other people's code to avoid issues. But if you want, you can say "Portions Copyright 2017 ..."

It would also be good to say that this is a modified version of statemate.c from (Malardalen website URL http:// ...).